examples/v1/pipelineruns/pipelinerun-with-final-tasks.yaml

```diff
@@ -0,0 +1,204 @@
+# Copied from https://github.com/tektoncd/catalog/blob/v1beta1/git/git-clone.yaml :(
+# This can be deleted after we add support to refer to the remote Task in a registry (Issue #1839) or
+# add support for referencing task in git directly (issue #2298)
```

Comment:
can this be changed? these features have been added

Reply:
I saw that you removed the code comment here, but what I'm suggesting is actually swapping to using the catalog task (pin the version) so this can serve as an example. Might need to wait on resolvers being enabled by default (part of RR beta PR)